configs/openmrs_config/concepts/labs.csv

```diff
@@ -5,6 +5,7 @@ f5f76f5e-2e6a-4112-ab4e-629299829ae3,,,Total Cholestrol,កូលេស្តេ
 c423325e-be95-42e2-ae38-b940c2586327,,,Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),0,90,mg/g,,,Test,Text,,,,
 07015405-c865-4a22-8ad1-c8863d5f005b,,,Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),0,9,mmol/L,,,Test,Text,,,,
 a8af7520-1350-11df-a1f1-0026b9348838,,,HBA1c,អេម៉ូក្លូប៊ីនអេវ័នស៊ី,HBA1c,អេម៉ូក្លូប៊ីនអេវ័នស៊ី,0,20,%,,,Test,Numeric,,,,
+163594AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,OGTT,តេស្តកម្រិតជាតិស្ករក្នុងឈាម២ម៉ោងក្រោយផឹកទឹកស្ករ៧៥ក្រាម,OGTT,តេស្តកម្រិតជាតិស្ករក្នុងឈាម២ម៉ោងក្រោយផឹកទឹកស្ករ៧៥ក្រាម,20,450,mg/dl,,,Test,Numeric,,,,
```

Comment:
@kazlaw shouldn't we add HBA1c as well?

Reply:
@kdaud  HBA1c already exists under this Uuid `a8af7520-1350-11df-a1f1-0026b9348838` in the file